github/PullRequestMergeStatus.py

```diff
@@ -54,10 +54,18 @@ def sha(self):
         """
         return self._sha.value
 
+    @property
```

Comment:
I do not see a PullRequestMergeStatus "squash" property in https://developer.github.com/v3/pulls/ or in the responses I see from github today. When does a PullRequestMergeStatus receive a "squash" property? Or is this wrong/outdated?